mbuild/tests/test_coordinate_transform.py

```diff
@@ -367,3 +367,20 @@ def test_spin_away_from_origin(self):
                 dtype=np.float)
         spun_points = _spin(points, np.pi/2, [0, 0, 1])
         assert np.allclose(spun_points, new_points_should_be, atol=1e-15)
+    def test_xyz_axis_transform(self):
+        rot_by_compound = mb.Compound(name = 'rot_by_compound')
```

Comment:
Super minor but by the style guide, there shouldn't be spaces around `=` when assigning attributes.

See here for a fun read: https://www.python.org/dev/peps/pep-0008/